cfn/README-GD.md

```diff
@@ -72,6 +76,65 @@ curl -s -X GET -H "x-aims-auth-token: $AL_TOKEN" https://api.global-services.glo
+```
+An example of a successful response is:
+
+```
+@{access_key_id=712c0b413eef41f6; secret_key=1234567890b3eea8880d292fb31aa96902242a076d3d0e320cc036eb51bf25ad}
+```
+
+**Important:** If the command returns error about not having necessary role, verify your Alert Logic account has administrator permissions. Click [here](https://console.cloudinsight.alertlogic.com/api/aims/) for more information about AIMS APIs.
+
+Make a note of the `access_key_id` and `secret_key` values, which you need when you deploy the CloudFormation template to your AWS account.
+
+**Note:** Each user can create only five access keys. If a "limit exceeded" response appears, you must delete one or more access keys before you can create new keys.
+
+**Type the following command to list access keys:**
+```
+Invoke-RestMethod -Method Get -Headers @{"x-aims-auth-token"=$AL_TOKEN} -Uri https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys
+```
+
+**Use the selected access_key_id in the following curl command to delete the key:**
+
+```
+Invoke-RestMethod -Method Delete -Headers @{"x-aims-auth-token"=$AL_TOKEN} -Uri https://api.global-services.global.alertlogic.com/aims/v1/$AL_ACCOUNT_ID/users/$AL_USER_ID/access_keys/<ACCESS_KEY_ID_HERE>
+```
+
+
+#### PowerShell < 3.0
+
+If you don't have PowerShell >= 3.0 installed, we suggest to [upgrade it](https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-windows-powershell#upgrading-existing-windows-powershell) to any version which is >= 3.0 or use command line and windows versions of [curl](https://curl.haxx.se/download.html) and [jq](https://stedolan.github.io/jq/download/) and follow the [instruction for Unix](#unix).
```

Comment:
Unfortunately, Windows 7 goes with PS 2.0 installed by default ([proof](https://4sysops.com/archives/powershell-versions-and-their-windows-version/)) which does not support `Invoke-RestMethod` / `Invoke-WebRequest` cmdlet. 

There are ways to work around it ([example on stackoverflow](https://stackoverflow.com/questions/25120703/invoke-webrequest-equivalent-in-powershell-v2)) but it does not really look nice & right way, so I'm not sure if we should really go for that one.

Alternate way which is described here is to upgrade to Windows 7 to have PS 3.0 or further, all links are available on MS website (https://docs.microsoft.com/en-us/powershell/scripting/setup/installing-windows-powershell#upgrading-existing-windows-powershell)

Reply:
@mtrybulec @kkuzmin do you guys think it's ok to ask to install PS 3.0, would it be convenient for customers?

Reply:
Typo: should be Windows, not windows.

I would think it's ok; earlier version are quite old; still, perhaps run this by Product or at the scrum of scrums.